v2/pages/adminAssessment/adminAssessment.sql

```diff
@@ -23,6 +23,44 @@ WHERE
     AND q.deleted_at IS NULL
 ORDER BY z.number, z.id, aq.number;
 
+-- BLOCK question_stats
+SELECT
+    sub.question_id,
+    avg(sub.mean_score_per_user_per_question) as mean_score_per_question,
+    sub.number,
+    corr(sub.mean_score_per_user_per_question, sub2.average_assessment_score_for_user) as discrimination
```

Comment:
Cap values to [0, 100] to avoid plotting outside the plot range. This can be done with the `greatest(0, least(100, ...))` pattern.